File: webclient/src/model/ImageDto.ts
import { FileUpdate } from "@/enum/FileUpdate";

export default interface IImageDto {
  id?: number;
  name: string;
  dateTime: Date;
  creationTime: Date;
  fileSize: number;
  photo: number[];
  photoPath?: string;
  lastPhotoName?: string;
  fileUpdate?: FileUpdate;
}

File: ImageUploader.DesktopCommon/Models/ImageDto.cs
using System;
using ImageUploader.DesktopCommon.Enum;

namespace ImageUploader.DesktopCommon.Models
{
    public class ImageDto : BaseFile
    {
        public string Name { get; set; }
        public DateTimeOffset DateTime { get; set; }
        public DateTimeOffset CreationTime { get; set; }
        public double FileSize { get; set; }
        public byte[] Photo { get; set; }
        public string PhotoPath { get; set; }
        public string LastPhotoName { get; set; }
        public FileUpdate FileUpdate { get; set; }
    }
}